big_tests/tests/mam_helper.erl

```diff
@@ -1011,7 +1011,7 @@ nick_to_jid(UserName, Config) when is_atom(UserName) ->
 make_jid(U, S, R) ->
     mongoose_helper:make_jid(U, S, R).
 
--spec backend() -> rdbms | riak | cassandra | false.
```

Comment:
it would never return false.

```erlang
determine_backend(_, []) ->
    disabled;
determine_backend(Host, [F | Rest]) ->
    case F(Host) of
        false -> % Because it is matched here
            determine_backend(Host, Rest);
        Result ->
            Result
    end.
 ```